.gitignore

```diff
@@ -9,4 +9,11 @@ cmake-*
 # put here local env stuff
 local
 
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# build directory
+build
+
+# clang-format and clang-tidy binaries
+clang-format
+clang-tidy
```

Comment:
- why would anyone have clang-* binaries in project root ? 
- for build directories we are using cmake-* naming convetion, so for example debug goes to cmake-build-debug etc...

Reply:
1) Because build scripts on macos downloads that binaries to project dir, and I added such option to conan too.
2) Okay, I don't know that before, so I can change it